src/decoder/mod.rs

```diff
@@ -302,18 +352,18 @@ impl<R: Read + Seek> Decoder<R> {
     pub fn colortype(&mut self) -> TiffResult<ColorType> {
         match self.photometric_interpretation {
             // TODO: catch also [ 8, 8, 8, _] this does not work due to a bug in rust atm
```

Comment:
This TODO can be removed now right?

Reply:
Wasn't exactly sure, almost sounds like the original author wanted to match formats with different alpha channel depth, but the line history is difficult to follow back into the image crate. Since it isn't clear what the TODO would be besides "support inhomogeneous bit depths" (which would be a big, crate-wide undertaking and not something local to this statement), I'll remove it.